nixos/tests/all-tests.nix

```diff
@@ -189,6 +189,7 @@ in {
   google-oslogin = handleTest ./google-oslogin {};
   gotify-server = handleTest ./gotify-server.nix {};
   grafana = handleTest ./grafana.nix {};
+  grafana-agent = handleTest ./grafana-agent.nix {};
```

Comment:
Please also add a passthru.tests in the grafana-agent package so that ofBorg is picking this up.

Reply:
Good idea